samples/spring-security-saml-dsl-sample/src/main/resources/application.yml

```diff
@@ -2,6 +2,10 @@ server:
   port: 8443
   ssl:
     enabled: true
-    key-alias: colombia
-    key-store: src/main/resources/saml/colombia.jks
-    key-store-password: colombia-password
\ No newline at end of file
+    key-alias: spring
+    key-store: src/main/resources/saml/keystore.jks
+    key-store-password: secret
+
+security:
+  saml2:
+      metadata-url: https://dev-348145.oktapreview.com/app/exk6bmtlqpOBRo4a20h7/sso/saml/metadata
```

Comment:
@rwinch Due to this being pointed at your own okta dev preview, I imagine that these tests would fail elsewhere (including CI)?